components/ButtonBar/src/MDCButtonBar.m

```diff
@@ -291,12 +291,21 @@ - (void)observeValueForKeyPath:(NSString *)keyPath
             [button setValue:newValue forKey:keyPath];
           }
 
-        } else if ([keyPath isEqualToString:NSStringFromSelector(@selector(title))]) {
-          [button setTitle:newValue forState:UIControlStateNormal];
+        } else if ([keyPath isEqualToString:NSStringFromSelector(@selector(accessibilityLabel))]) {
+          button.accessibilityLabel = newValue;
 
         } else if ([keyPath isEqualToString:NSStringFromSelector(@selector(image))]) {
           [button setImage:newValue forState:UIControlStateNormal];
 
+        } else if ([keyPath isEqualToString:NSStringFromSelector(@selector(tag))]) {
+          button.tag = [newValue integerValue];
+
```

Comment:
Nit: Extra newline here and below

Reply:
This is the format of this file already. I can address the whitespace in a follow-up change.
https://github.com/material-components/material-components-ios/issues/4381